### PR TITLE
Providing specific examples and clearing up the script docs

### PR DESCRIPTION
**Pull request by vladimp**
1. Providing guidance on when the PSF can help with app compat issues
2. Clearing up the scripts docs as they moved to msdn
3. Clarifying how to get the PSF binaries from NuGet.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+	And about the available configurations [here] or in the Documention section below (https://github.com/microsoft/MSIX-PackageSupportFramework/tree/master/PsfLauncher)
+
+We are looking for your feedback, please file any issues or feedback in the [Issues section] (https://github.com/Microsoft/MSIX-PackageSupportFramework/issues) or in our [tech community] (https://techcommunity.microsoft.com/t5/Package-Support-Framework/bd-p/Package-Support)
 
 ## Documentation
-Check out our [step by step guide](https://docs.microsoft.com/en-us/windows/uwp/porting/package-support-framework), it will walk you through the main PSF workflows and provides the key documentation.
+Check out our [step by step guide](https://docs.microsoft.com/en-us/windows/uwp/porting/package-support-framework), it will walk you through the main PSF workflows and provides the key documentation. Check out the [PSF scripts support] (https://docs.microsoft.com/en-us/windows/msix/psf/run-scripts-with-package-support-framework) that we recently added to our framework, it provides great flexibility to yuour packages.
```
Comment (jolonghi) [inline README.md]: >yuour [](start = 405, length = 5)

your

File: README.md [hunk 2]
```diff
@@ -1,17 +1,33 @@
...
 
 See also:
 * [Package Support Framework package layout](layout.md)
 * [Package Support Framework Nuget package install](https://www.nuget.org/packages/Microsoft.PackageSupportFramework)
 * [Instructions for authoring your own shim dll](Authoring.md)
```
Comment (jolonghi) [inline README.md]: >shim [](start = 39, length = 4)

fixup

File: README.md [hunk 3]
```diff
@@ -1,17 +1,33 @@
 # Package Support Framework
-This project provides tools, libraries, documentation and samples for creating app-compat fixups to enable classic Win32 applications to be distributed and executed as packaged apps.
+This project provides tools, libraries, documentation and samples for creating app-compat fixups to enable classic Win32 applications to be distributed and executed as MSIX packaged apps.
+
+Here are some common examples where you can find the PSF useful:
+	1. Your app can't find some .dlls when launched, you may need to set your current working directory.
+	You can learn about the required current working directory in the original shortcut before you converted to MSIX.
+	2. The app writes into the install folder. You will typically see it by "Access Denied" errors in
```
Comment (dhoehna) [inline README.md]: Can we have a link to what an ACCESS DENIED error looks like in procmon?

Comment (vladimp) [reply README.md]: Sure will do

File: README.md [hunk 4]
```diff
@@ -1,16 +1,30 @@
 # Package Support Framework
-This project provides tools, libraries, documentation and samples for creating app-compat fixups to enable classic Win32 applications to be distributed and executed as packaged apps.
+This project provides tools, libraries, documentation and samples for creating app-compat fixups to enable classic Win32 applications to be distributed and executed as MSIX packaged apps.
+
+Here are some common examples where you can find the PSF useful:<br/>
+	1. Your app can't find some .dlls when launched, you may need to set your current working directory.
```
Comment (dhoehna) [inline README.md]: Remove the tabs from 1., 2., and 3. and add a line break before 1. to make this look like a list.  You can use https://stackedit.io/app# to see what the readme will look like.

**Comment by dhoehna** [comment]
Looks good.  Added one comment on how to make the list look like a list.

Can you please change the PR target to the develop branch?  There should not be any MErge conflicts because you are changing the README.md file.

File: README.md (hunk 2 above)
Comment (jolonghi) [reply README.md]: Should use term fixup

---
In reply to: [337138171](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/96#discussion_r337138171) [](ancestors = 337138171)

**Pull request merged**